ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/ApicurioSteps.java

```diff
@@ -324,4 +326,27 @@ public void switchContextOn() {
     public void switchContextOff() {
         Selenide.switchTo().parentFrame();
     }
+
+    @When("^click on the \"([^\"]*)\" apicurio button*$")
```

Comment:
This seems to never be used

Reply:
finally no, but this could be handy, but this could be handy when button that need to be clicked is part of "apicurio main element". Consider to remain it for the future.